app/shared/components/header/header.component.html

```diff
@@ -0,0 +1,20 @@
+<div id="primary-menu">
+  <div class="menu-container clearfix flat-menu">
+    <div id="logo">
+      <a [routerLink]="['Root']" title="Gapminder">
+        <img class="logo"
+             src="http://www.gapminder.org/wp-content/themes/gapminder/images/logo.png"
```

Comment:
Shouldn't we serve all the content from contentful including images? Or we can rely on http://www.gapminder.org/wp-content as well?